test/libsolidity/syntaxTests/using/library_functions_attached_in_single_directive_inside_library.sol

```diff
@@ -0,0 +1,19 @@
+library L {
+    function publicFunction(uint) public pure {}
+    function externalFunction(uint) external pure {}
+    function internalFunction(uint) internal pure {}
+    function privateFunction(uint) private pure {}
+
+    using {publicFunction, externalFunction, internalFunction, privateFunction} for uint;
```

Comment:
Please include `unqualified` somewhere in the test name. That's an important detail.